docs/changelog_for_devs.md

```diff
@@ -21,6 +21,31 @@ APIs/RPC interface.
 
 - ⚠️ Add `outsider` field to `MarketBonds` struct. In particular, the `Market`
   struct's layout has changed ([#903]).
+  # v0.3.9
+
+All things about Global Disputes:
+- Added new dispatchable function:
+  - `refund_vote_fees` - Return all vote funds and fees, when a global dispute
+    was destroyed.
+- Added the following events:
+  - `OutcomeOwnerRewarded` for `Possession::Paid`
+  - `OutcomeOwnersRewarded` for `Possession::Shared`
+  - `OutcomesFullyCleaned` and `OutcomesPartiallyCleaned` for extrinsic `refund_vote_fees`
+- Removed the following events:
+- Added enum `Possession` with variants:
+  - `Paid { owner: AccountId, fee: Balance }`
+  - `Shared { owners: BoundedVec }`
+- `OutcomeInfo` has the following fields:
```

Comment:
I think @saboonikhil would appreciate some info on how the indexer should read global dispute data from the chain.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/912/commits/685b5d32f7ed5e83186e7c84d3720a4e1d124fa3